ministryofjustice/laa-claim-non-standard-magistrate-fee-backend
# Add Sentry to Application

## Description of change

- Add Sentry to application initialiser
- Add test error to verify that Sentry is working (another PR is needed to remove this after testing)

## Link to relevant ticket
[Add Sentry to Application
](https://dsdmoj.atlassian.net/browse/CRM457-158)


## How to manually test the feature:

- Go to MOJ Sentry and login
- View Issues
- There should be an issue with title "test message"
